packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default {
+    name: "utility-plugins-remove-vowels",
+    type: "utility-plugins",
+    removeVowels: str => {
+        return str.replace(/[aeiouAEIOU]/g, "");
+    }
+};
```

Comment:
This whole file can just be this:
```
export default () => {
    // TODO: implement your package
}
```

It's just a skeleton for a generic node package, the goal is to have the configs in place, which you did. Let the developer bother with implementation.

Reply:
done